cypress/support/fragments/user/loans.js

```diff
@@ -0,0 +1,82 @@
+import {
+  MultiColumnList,
+  MultiColumnListCell,
+  Modal,
+  Dropdown,
+  Button,
+  TextArea,
+} from '../../../../interactors';
+import { ITEM_STATUSES } from '../../../support/constants';
+
+const LOANS_HISTORY_LIST_ID = 'list-loanshistory';
+const STATUS_COLUMN_INDEX = 2;
+const ACTIONS_COLUMN_INDEX = 10;
+const DECLARE_LOST_MODAL_TITLE = 'Confirm item status: Declared lost';
+const DECLARE_LOST_ACTION_NAME = 'Declare lost';
+
+export default {
+  getApi(userId) {
+    return cy.okapiRequest({
+      method: 'GET',
+      path: 'circulation/loans',
+      searchParams: {
+        query: `userId==${userId}`
+      },
+    }).then(({ body }) => body.loans);
+  },
+  getLoansHistoryList() {
+    return MultiColumnList(LOANS_HISTORY_LIST_ID);
+  },
+  getDeclareLostModal() {
```

Comment:
use interactor const with instead of extra getter

Reply:
Good point. Thanks. Done